packages/design/src/icons/getIcon.ts

```diff
@@ -0,0 +1,113 @@
+import classnames from "classnames";
+import styles from "./Icon.css";
+import sizes from "./Sizes.css";
+import colors from "./Colors.css";
+import { iconMap } from "./iconMap";
+
+export type IconNames =
+  | keyof typeof iconMap.icons
+  | "longArrowUp"
+  | "longArrowDown"
+  | "remove"
+  | "thumbsDown"
+  | "truck";
+export type IconColorNames =
```

Comment:
The reason we don't use `keyof typeof colors` here is that we use a global type declaration for all css files on jobber-mobile which prevents us from getting any meaningful types from css files.